taxi/test/test_form.py

```diff
@@ -0,0 +1,61 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.models import Manufacturer, Car, Driver
+
+
+class TestForm(TestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create_user(
+            username="tset",
+            password="test12345",
+        )
+
+        self.client.force_login(self.user)
+
+    def test_car_should_search_by_name(self):
+        manufacturer = Manufacturer.objects.create(
+            name="test",
+            country="test_country"
+        )
+
+        Car.objects.create(
+            model="test",
+            manufacturer=manufacturer
+        )
+        cars = Car.objects.all()
+        response = self.client.get(reverse("taxi:car-list") + "?model=test")
+
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, "taxi/car_list.html")
+        self.assertEqual(list(response.context["car_list"]), list(cars))

```

Comment:
what if search feature won't work ? will this test check this ? 
same for other tests

Reply:
u didn't fix that, add more cars with different model names